pom.xml

```diff
@@ -140,13 +140,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>findbugs-maven-plugin</artifactId>
-        <configuration>
-          <excludeFilterFile>src/findbugs/excludesFilter.xml</excludeFilterFile>
-        </configuration>
-      </plugin>
       <plugin>
         <groupId>org.antlr</groupId>
         <artifactId>antlr4-maven-plugin</artifactId>
```

Comment:
Apparently there is a “Credentials Query Language” from #55 (no code review or downstream PRs with usage examples). @stephenc?

----

```java
(instanceof com.cloudbees.plugins.credentials.common.UsernameCredentials) && !(username == "bob")
```

----

This walk-through is incomplete and needs the following additions:

* [ ] …
* [ ] Provide some details on how the Credentials Query Language can be used to limit querying credentials from the remote service

Reply:
Yes there is a query language designed to allow remote credentials stores to optimise search scope to limit trawling through all the credentials.